lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/api/ResourceRequestEvent.java

```diff
@@ -35,6 +36,11 @@
 public interface ResourceRequestEvent {
 
   /**
+   * @return The node labels.
+   */
+  Map<String, String> getNodeLabels();
```

Comment:
Same as everywhere: Please document why this is a `Map`, and what key and value represent.